lib/author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_reader :first_name, :last_name, :title, :publication_year, :name, :books
+
+  def initialize(author)
+    @first_name = author[:first_name]
+    @last_name = author[:last_name]
+    @name = author[:first_name] + " #{author[:last_name]}"
+    @books = []
+  end
+
+  def write(title, date)
```

Comment:
This is sooooo close - you just need to add `@books.last` to the last line (line 13) for this method to be perfect.